identifierresolution.grace

```diff
@@ -239,7 +239,7 @@ class newScopeIn(parent') kind(variety') {
                 mem := ast.memberNode.new("outer", mem) scope(self)
             }
         }
-        errormessages.syntaxError "no method {name}"
+        errormessages.syntaxError "no method {errormessages.canonicalMethodName(name)}"
```

Comment:
Here and at line 261, you are inside a method that takes an AstNode `node` as parameter.
So you can access the canonical name directly as `node.canonicalName`; there is no need to convert the numeric string to canonical form